content/laptop-battery-thresholds.md

```diff
@@ -16,11 +16,13 @@ tableOfContents: true
 
 Charging thresholds allow your System76 laptop to avoid charging the battery until it has dropped below a lower bound (the start threshold), and to stop charging when it reaches an upper bound (the end threshold). This is useful when your laptop is plugged into an AC power adapter for extended periods of time, as it prevents unnecessary micro-charging that would reduce battery longevity.
 
-To determine if your laptop has Open Firmware or proprietary firmware, see [this article](/articles/open-firmware-systems). (If a system has Open Firmware, then it must also have Open EC to work with charging thresholds.) See [Charging Thresholds](#configuring-charging-thresholds-open-firmware) for Open Firmware systems or [FlexiCharger](#configuring-flexicharger-proprietary-firmware) for proprietary firmware systems.
+The default charging profile for System76 laptops with Open Firmware have a start threshold of 90 and a stop threshold of 100. This increases battery longevity, while keeping the battery very close to a full charge.
+
+To determine if your laptop has Open Firmware or proprietary firmware, see [this article](/articles/open-firmware-systems). (If a system has Open Firmware, then it must also have Open EC to work with charging thresholds). See [Charging Thresholds](#configuring-charging-thresholds-open-firmware) for Open Firmware systems or [FlexiCharger](#configuring-flexicharger-proprietary-firmware) for proprietary firmware systems.
```

Comment:
In this case, the period actually should be inside of the parenthesis (as it was) because it's a full sentence inside of the parenthesis. It can get stylistically subjective when it comes to longer or independent clauses, but there's not really any room for interpretation on this line.

Reply:
I see. Is having an entire sentence in parens correct? I'm not sure that's something I've ever seen anywhere else.

Reply:
Did some research, and I guess it's correct. I'll change it.

Reply:
It's grammatically valid.

https://www.yourdictionary.com/articles/punctuation-inside-outside-parentheses
https://grammarist.com/punctuation/period-before-or-after-parentheses/
https://www.utsa.edu/twc/documents/Parentheses.pdf

Reply:
Changed it back, and learned something new today.